src/ISFirmwareUpdater.cpp

```diff
@@ -519,20 +519,20 @@ void ISFirmwareUpdater::runCommand(std::string cmd) {
             // any target which doesn't report version info will also expect the old MD5 digest
             if (!target_devInfo) {
                 // TODO: We should be able to remove most of this after 2.1.0 has been released
-                if ( ((target & fwUpdate::TARGET_IMX5) && (devInfo->hardware == HDW_TYPE__IMX)) ||
-                     ((target & fwUpdate::TARGET_GPX1) && (devInfo->hardware == HDW_TYPE__GPX)) ) {
+                if ( ((target & fwUpdate::TARGET_IMX5) && (devInfo->hardwareType == DEV_INFO_HARDWARE_TYPE_IMX)) ||
```

Comment:
Since we are consolidating these, is there still a reason to denote "DEV_INFO_HARDWARE_TYPE_IMX" as opposed to just "HARDWARE_TYPE_IMX"?    Personally, I'm a fan of the more abbreviated HDW_TYPE__GPX, but don't think we need the "DEV_INFO" since its not specific do DEV_INFO anymore.